src/style.css

```diff
@@ -2,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  padding: 12px 0;
+}
+
+.search-bar {
+  display: block;
+  width: 100%;
+  box-sizing: border-box;
+  border-radius: 4px;
+  font-weight: 300;
+  transition: border-color 0.3s ease;
+  outline: none;
+  font-size: 16px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  font-family: inherit;
+}
+
+input::placeholder {
+  color: #3d4e61;
+  font-weight: normal;
+}
+
+[data-qa='big'] input {
+  height: 70px;
+  padding-left: 62px;
+  box-sizing: border-box;
+  background-size: 19px;
+  background-position:
+    25px center,
+    0 0;
+}
+
+[data-qa='small'] input {
+  height: 42px;
+  margin-top: 20px;
+  background-position: 12px center;
+  background-size: 11px;
+  box-sizing: border-box;
+  font-size: 14px;
+  font-weight: bold;
+  padding-left: 33px;
+}
+
+.search-bar:focus {
+  box-shadow: 0 4px 4px 0 #00000040;
+  outline: none;
+  background:
+    url(images/Search.svg) 25px center no-repeat,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+}
+
+.search-bar:hover {
+  border-color: #888;
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.icon {
+  background: url(images/Search.svg) 25px center no-repeat;
```

Comment:
Ensure that the image 'Search.svg' exists in the 'images' directory. If the image file is not found, the background image for the icon will not display correctly.